### ISSUE TITLE
How to get clean text in mix mode ?

### ISSUE THREAD

**Issue by manojprabhut-CC** [yairEO/tagify#964]
For example i have content like "[[{"value":"1","text":"John Henry","title":"John Henry","prefix":"@"}]]  keep follow on  " which i need to get it like "John Henry keep follow on". I have implemented mvc .net application with mix mode.

**Comment by yairEO** [comment]
This is basic DOM and not something Tagify provides (as a library):

You access the Tagify DOM element and reads the `textValue`

![image](https://user-images.githubusercontent.com/845031/148381266-f3085e35-e47b-4afb-bec5-004ddcd716af.png)

**Comment by manojprabhut-CC** [comment]
![image](https://user-images.githubusercontent.com/69341332/148382038-72e8b44b-2b7c-4708-8623-0cb98d502927.png)
![image](https://user-images.githubusercontent.com/69341332/148382073-98acab33-570e-4f83-9652-961bcf7b428a.png)

This is what happens when i had used "tagify.DOM.scope.textContent", I had typed 1st two letters and selected the value from dropdown.

**Comment by yairEO** [comment]
Are you asking a question?

**Comment by manojprabhut-CC** [comment]
Yes, @yairEO . I have tried the mentioned scenario like when i type "@Jo" tagify loads the drop down and i had selected "John Shelby" on this case i'm getting "@jo" in tagify.DOM.scope.textContent.

**Comment by yairEO** [comment]
I cannot help without seeing a jsbin demo with your exact configuration (like I've asked in the issues template when you opened this issue)
